From 6a33823824f1ec4152514ee0cb4506ebdc8fce26 Mon Sep 17 00:00:00 2001
From: Ajay Gautam <ajay.gautam@amlogic.com>
Date: Tue, 25 May 2021 11:54:34 -0700
Subject: [PATCH 4/6] RDK: disable continuous print from hdmi

PD#SWPL-46838

Problem:
hdmi print "find SD only 4x3" on console is flooding after
hdcp22 key is enabled

Solution:
change this print from info to debug to avoid this console
flooding print and enabled it for debug only

Verify:
RDK_Firebolt

Change-Id: Ia5a2ac5c13cc212973465a29f87c58673434b383
Signed-off-by: Ajay Gautam <ajay.gautam@amlogic.com>
---
 drivers/amlogic/media/vout/hdmitx/hdmi_tx_20/hdmi_tx_edid.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/amlogic/media/vout/hdmitx/hdmi_tx_20/hdmi_tx_edid.c b/drivers/amlogic/media/vout/hdmitx/hdmi_tx_20/hdmi_tx_edid.c
index 27dead5d756b..3b814f15420e 100644
--- a/drivers/amlogic/media/vout/hdmitx/hdmi_tx_20/hdmi_tx_edid.c
+++ b/drivers/amlogic/media/vout/hdmitx/hdmi_tx_20/hdmi_tx_edid.c
@@ -3013,7 +3013,7 @@ enum hdmi_vic hdmitx_edid_get_VIC(struct hdmitx_dev *hdev,
 		}
 		/* if TV only supports 480p/2, add 480p60hz as well */
 		if (is_sink_only_sd_4x3(hdev, disp_mode, &vic)) {
-			pr_info("hdmitx: find SD only 4x3\n");
+			pr_debug("hdmitx: find SD only 4x3\n");
 		}
 	}
 	if (vic == HDMI_UNKNOWN && vesa_vic != HDMI_UNKNOWN) {
-- 
2.29.2

